spikely/pipeman/pipeman.py

```diff
@@ -1,4 +1,6 @@
 import sys
+from ctypes import cdll
+os_encoding = 'cp' + str(cdll.kernel32.GetACP())
```

Comment:
Will this work on a Linux system?

Reply:
My bad, it won't. I just added another if/else.